### PR TITLE
Add changes related to new SMPP inbound endpoint UI schema 

### PR DESCRIPTION
**Pull request by AmilaSamith**

### CODE REVIEW

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 1]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
-> - **systemId** : username to access the SMSC
-> - **password** : password to access the SMSC 
+3. Click `+` button next to `Inbound Endpoints` and select `SMPP` to add a new **SMPP inbound endpoint**.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-create-new-inbound-endpoint.png" title="Creating smpp inbound endpoint" width="1000" alt="Creating smpp inbound endpoint" style="border:1px solid black"/>
+    <br/>
+
+4. Configure the SMPP inbound endpoint as shown below and click `Create`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-config-endpoint-1.png" title="Configure SMPP inbound endpoint 1" width="1000" alt="Configure SMPP inbound endpoint 1" style="border:1px solid black"/>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:55]:
```suggestion
    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-config-endpoint-1.png" title="Configure SMPP inbound endpoint 1" width="1000" alt="Configure SMPP inbound endpoint 1"/>
```

Please remove additional styling from all the images.

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 2]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+    <br/>
+   
+    !!! Note
+        When creating the SMPP inbound endpoint you have two options in defining the injecting sequence and error sequence.
+        <br/>- **Automatic** : Click the checkbox *Automatically generate sequence*
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:60]:
```suggestion
        <br/>- **Automatic**: Click the *Automatically generate sequence* checkbox
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 3]
```diff
@@ -105,7 +113,12 @@ You can download the ZIP file and extract the contents to get the project code.
 1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `SMPP Connector`. Click on `SMPP Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. Copy this .jar file into **PRODUCT-HOME/lib** folder. 
 
 2. Download  [jsmpp-2.1.0-RELEASE.jar](https://mvnrepository.com/artifact/com.googlecode.jsmpp/jsmpp/2.1.0-RELEASE) and [asyncretry-jdk7-0.0.6.jar](https://mvnrepository.com/artifact/com.nurkiewicz.asyncretry/asyncretry-jdk7/0.0.6) copy inside **PRODUCT-HOME/lib** folder.
-   
+
+    !!! Tip
+        Alternatively for deployment step 1 and 2, MI for VS Code has provided the capability of copying jars to **PRODUCT-HOME/lib** folder while exporting the capp. You can copy necessary jars to **deployment/libs** folder under project explorer view.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:118]:
```suggestion
        Alternatively, for deployment steps 1 and 2, MI for VS Code provides the capability to copy JARs to the `<PRODUCT_HOME>/lib` folder while exporting the CApp. You can copy the necessary JARs to the **deployment** > **libs** folder in the **Explorer** view.
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 4]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+            <parameter name="reconnectInterval">1000</parameter>
+            <parameter name="retryCount">3</parameter>
+            <parameter name="exponentialFactor">5</parameter>
+            <parameter name="maximumBackoffTime">10000</parameter>
+        </parameters>
+    </inboundEndpoint>
+    ```        
+
+!!! Note
+    To configure the `systemId` and `password` parameter value, please use the steps given under the topic `Configure the SMSC (Short Message Service Center) simulator` in the [Setting up the SMPP Connector ]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:92]:
```suggestion
    To configure the `systemId` and `password` parameter values, see [Configure the SMSC]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/#configure-the-smsc-short-message-service-center-simulator).
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 5]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+    <br/>- **systemId** : username to access the SMSC
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:93]:
```suggestion
    <br/>- **systemId**: The username to access the SMSC
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 6]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+    <br/>- **password** : password to access the SMSC
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:94]:
```suggestion
    <br/>- **password**: The password to access the SMSC
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md (hunk 1 above)
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 7]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+
+4. Configure the SMPP inbound endpoint as shown below and click `Create`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-config-endpoint-1.png" title="Configure SMPP inbound endpoint 1" width="1000" alt="Configure SMPP inbound endpoint 1" style="border:1px solid black"/>
+    <br/>
+   
+    !!! Note
+        When creating the SMPP inbound endpoint you have two options in defining the injecting sequence and error sequence.
+        <br/>- **Automatic** : Click the checkbox *Automatically generate sequence*
+        <br/>- **Manual** : You can select already defined sequences as injecting and error sequences.
```
[the remark this answers is not in the document]
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md (hunk 3 above)
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md (hunk 4 above)
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md (hunk 2 above)
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md (hunk 5 above)
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: Resolved by https://github.com/wso2/docs-mi/pull/1094/commits/c16e88e3402822423bda242f91707d64a80a3cd4

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 8]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+         <log level="full"/>
+      </sequence>
+      ```
    
-> **Note**: To configure the `systemId` and `password` parameter value, please use the steps given under the topic `Configure the SMSC (Short Message Service Center) simulator` in the [Setting up the SMPP Connector ]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation.
-> - **systemId** : username to access the SMSC
-> - **password** : password to access the SMSC 
+3. Click `+` button next to `Inbound Endpoints` and select `SMPP` to add a new **SMPP inbound endpoint**.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-create-new-inbound-endpoint.png" title="Creating smpp inbound endpoint" width="1000" alt="Creating smpp inbound endpoint" style="border:1px solid black"/>
```
[the remark this answers is not in the document]
Comment (AmilaSamith) [reply en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md]: Resolved by https://github.com/wso2/docs-mi/pull/1094/commits/bd323c792fc8c9853e9def595bfbd180207efb6b

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 9]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
    
-> **Note**: To configure the `systemId` and `password` parameter value, please use the steps given under the topic `Configure the SMSC (Short Message Service Center) simulator` in the [Setting up the SMPP Connector ]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation.
-> - **systemId** : username to access the SMSC
-> - **password** : password to access the SMSC 
+3. Click `+` button next to `Inbound Endpoints` and select `SMPP` to add a new **SMPP inbound endpoint**.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-create-new-inbound-endpoint.png" title="Creating smpp inbound endpoint" width="1000" alt="Creating smpp inbound endpoint"/>
+    <br/>
+
+4. Configure the SMPP inbound endpoint as shown below and click `Create`.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:53]:
```suggestion
4. Configure the SMPP inbound endpoint as shown below and click **Create**.
```

File: en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md [hunk 10]
```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
...
+                      expression="get-property('SMPP_ServiceType')"/>
+         </log>
+         <log level="full"/>
+      </sequence>
+      ```
    
-> **Note**: To configure the `systemId` and `password` parameter value, please use the steps given under the topic `Configure the SMSC (Short Message Service Center) simulator` in the [Setting up the SMPP Connector ]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation.
-> - **systemId** : username to access the SMSC
-> - **password** : password to access the SMSC 
+3. Click `+` button next to `Inbound Endpoints` and select `SMPP` to add a new **SMPP inbound endpoint**.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md:48]:
```suggestion
3. Click on the **+** button next to **Inbound Endpoints** and select **SMPP** to add a new SMPP inbound endpoint.  
```

**Review by DinithiDiaz — APPROVED** [review]

**Pull request merged**